android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -602,7 +799,7 @@ private void checkWhichCameraAsDefault(){
 		if (backCameraId == Integer.MIN_VALUE && frontCameraId != Integer.MIN_VALUE) {
 			TiCameraActivity.whichCamera = MediaModule.CAMERA_FRONT;
 		} else {
-			TiCameraActivity.whichCamera = MediaModule.CAMERA_REAR;
+			//TiCameraActivity.whichCamera = MediaModule.CAMERA_REAR;				
```

Comment:
I see and understand. In that case, please remove/comment out the else bracket as well if you are removing that portion of the logic.